app/overrides/views/admin_subscriptions_menu_link.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-if !Spree::Backend::Config.respond_to?(:menu_items)
+if !SolidusSubscriptions.configuration.disable_admin_menu_link && !Spree::Backend::Config.respond_to?(:menu_items)
```

Comment:
We can remove this override completely. `menu_items` is available in Solidus 2.9, which is the earliest Solidus version we currently support: https://solidus.io/security/.